src/mono/mono/mini/interp/transform.c

```diff
@@ -2504,6 +2504,19 @@ interp_handle_intrinsics (TransformData *td, MonoMethod *target_method, MonoClas
 		// and always return false for IsDynamicCodeCompiled
 		if (!strcmp (tm, "get_IsDynamicCodeCompiled"))
 			*op = MINT_LDC_I4_0;
+	} else if (in_corlib &&
+			!strncmp ("System.Runtime.Intrinsics.X86", klass_name_space, 29) &&
```

Comment:
Nit: `memcmp` might be faster

Reply:
length of `klass_name_space` is not known